src/HistoryService.Data/Interfaces/IServiceHistoryRepository.cs

```diff
@@ -0,0 +1,18 @@
+﻿using LT.DigitalOffice.HistoryService.Models.Db;
+using LT.DigitalOffice.HistoryService.Models.Dto.Requests.Filters;
+using LT.DigitalOffice.Kernel.Attributes;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.HistoryService.Data.Interfaces
+{
+  [AutoInject]
+    public interface IServiceHistoryRepository
+    {
+        Guid Create(DbServiceHistory dbServiceHistory);
+
+        bool DoesServiceHistoryVersionExist(string version, Guid id);
```

Comment:
you in ServiceHistory repo, i think, no need write what it in method name

Reply:
fixed